src/components/DetailComment/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@bcrs-shared-components/detail-comment",
-    "version": "1.0.3",
+    "version": "1.0.5",
```

Comment:
What rules should we implement for updating the version (or does Lerna do it for us)?

Here, the "patch" number is changes, not the minor version. I am wondering when should update the major/minor/patch versions... Thoughts?

Reply:
I missed to merge last version update(1.0.4 is published in npm before), so i choose next patch version. We can follow [npm standard](https://docs.npmjs.com/about-semantic-versioning)